src/host/CursorBlinker.cpp

```diff
@@ -28,6 +28,12 @@ void CursorBlinker::UpdateSystemMetrics()
 {
     // This can be -1 in a TS session
     _uCaretBlinkTime = ServiceLocator::LocateSystemConfigurationProvider()->GetCaretBlinkTime();
+
+    // If animations are disabled, or the blink rate is infinite, blinking is not allowed.
+    BOOL animationsEnabled = true;

```

Comment:
I _believe_ that this wants to be `TRUE` or `YES` or something that matches `loud BOOL`